### PR TITLE
feat: Add Unbond and Withdraw Sections to Transaction Dialogs

### PR DESCRIPTION
**Pull request by izikdepth**

### CODE REVIEW

**Comment by b00f** [comment]
The pictures are old; the new one has a memo field. Also, make sure there is no background in the screenshot. Try to capture the dialog in the screenshot.

**Comment by izikdepth** [comment]
> The pictures are old; the new one has a memo field. Also, make sure there is no background in the screenshot. Try to capture the dialog in the screenshot.

done. thank you

File: website/_i18n/en/user-guides/send-transactions-gui/index.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ click on the "Send" button to sign and broadcast the transaction to the Pactus b
 It's important to note that the [transaction fee]({{ site.baseurl }}/learn/transaction/fee/)
 will be calculated based on the amount you're sending.
 
+Also, when sending or receiving a transaction from either exchange or another person that's not you,
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/send-transactions-gui/index.md:33]: When sending (remove also)

**Review by Mr-HoDL58 — APPROVED** [review]

File: website/_i18n/en/user-guides/send-transactions-gui/index.md (hunk 1 above)
Comment (izikdepth) [reply website/_i18n/en/user-guides/send-transactions-gui/index.md]: done. thanks

**Pull request merged**